apps/postgresql.go

```diff
@@ -130,13 +131,27 @@ func (p LoggingProcessorPostgresql) Components(ctx context.Context, tag string,
 						},
 					},
 				},
+				{
+					// This parser matches the default log_line_prefix from SLES12 in our testing
+					// log_line_prefix = '%m %d %u [%p]'
+					// Sample line: 2024-05-30 15:34:26.572 UTC postgres postgres [23958]STATEMENT:  INSERT INTO
+					//					test2 (id) VALUES('1970-01-01 00:00:00.123 UTC');
+					Regex: `^(?<time>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}.\d{3,} \w+)\s*(?:\s+(?<database>\S*)\s+(?<user>\S*))?\s*\[(?<tid>\d+)\]\s*(?<level>\w+):\s+(?<message>[\s\S]*)`,
```

Comment:
[Optional] Both of these regexes treat the user/database pair as optional, so I wonder if it makes sense to combine them into one and avoid adding a new parser…

Reply:
I was concerned that fluentbit might not support two different capture groups with the same name, but didn't find anything in the documentation. Also concerned if it's not documented about the potential that it could change in the future.

Reply:
You're right, this may not work in fluent-bit.